app/api/groups/create_user_batch.feature

```diff
@@ -98,10 +98,10 @@ Feature: Create a user batch
       | group_prefix | custom_prefix | size | creator_id | created_at          |
       | test         | custom        | 2    | 21         | 2019-07-16 22:02:28 |
     And the table "users" should be:
-      | group_id            | latest_login_at | latest_activity_at | temp_user | registered_at       | login_id  | login           | email | first_name  | last_name |
-      | 21                  | null            | null               | 0         | null                | null      | owner           | null  | Jean-Michel | Blanquer  |
-      | 5577006791947779410 | null            | null               | 0         | 2019-07-16 22:02:28 | 100000029 | test_custom_jzk | null  | null        | null      |
-      | 8674665223082153551 | null            | null               | 0         | 2019-07-16 22:02:28 | 100000030 | test_custom_ctc | null  | null        | null      |
+      | group_id            | latest_login_at | latest_activity_at | temp_user | registered_at       | login_id  | login           | default_language | email | first_name  | last_name |
+      | 21                  | null            | null               | 0         | null                | null      | owner           | fr               | null  | Jean-Michel | Blanquer  |
```

Comment:
as "fr" is the default anyway, probably that's not a good choice to demonstrate that it is copied from the creator

Reply:
fixed